lua/nvim-treesitter/query.lua

```diff
@@ -71,4 +91,6 @@ function M.iter_prepared_matches(query, qnode, bufnr, start_row, end_row)
   end
 end
 
+
+
```

Comment:
great stuff :) Can you remove these ?

Reply:
Ooops. wrong PR with an outdated commit. That happens during rebase. Also fixed this in #63